### PR TITLE
Move a couple Mixer docs

### PR DESCRIPTION
**Pull request by geeknoid**

### CODE REVIEW

File: mixer/v1/service.proto [hunk 1]
```diff
@@ -14,12 +14,6 @@
 
 syntax = "proto3";
 
-// $title: Mixer
-// $description: API definitions to interact with Mixer
-// $location: https://istio.io/docs/reference/api/istio.mixer.v1.html
-// $front_matter: redirect_from: /docs/reference/api/mixer/mixer.html
-
-
```
Comment (guptasu) [inline mixer/v1/service.proto]: We don't want to document Mixer API ?

**Comment by geeknoid** [comment]
Nope, this doesn't belong on istio.io. istio.io is for operators and not
developers.

**Comment by guptasu** [comment]
Agree, just wanted to make sure this is not by accident.

**Pull request merged**